vanetza/btp/data_indication.hpp

```diff
@@ -34,6 +35,7 @@ struct DataIndication
     geonet::LongPositionVector source_position;
     geonet::TrafficClass traffic_class;
     boost::optional<geonet::Lifetime> remaining_packet_lifetime;
+    boost::optional<unsigned> remaining_hop_limit;
```

Comment:
I suppose `remaining_packet_lifetime` and `remaining_hop_limit` are optional to implement but always available from a GN protocol perspective. What do you think about making these two fields "mandatory" in Vanetza then?